src/models/Brand.ts

```diff
@@ -0,0 +1,19 @@
+import mongoose from 'mongoose';
+
+export type IBrandUpdate = Partial<IBrand>;
+
+export interface IBrand {
+    logoImgUrl: string;
+    backgroundImgUrl: string;
+    poolAddress: string;
+    clientId: string;
+}
+
+const brandSchema = new mongoose.Schema({
+    logoImgUrl: String,
+    backgroundImgUrl: String,
+    poolAddress: String,
```

Comment:
@GarfDev I have a task open to move away from using the `poolAddress` as an identifier for the pool, since it is not necessarily unique and could return a different contract when called for a different chainId. 

Do you think it is possible to use the poolId here? That would be better since the `mongoId` in `AssetPool._id` is actually unique😅

I guess when building the reward links or QR codes we need to also include `poolId` as a parameter, so auth can pick it up and use it in its brand proxy request.

Reply:
It based on what data rewardData can contain in Auth app, I will try to check if I can get Pool ID from there

Reply:
It needs to be set in there in the first place I guess when generating the QR and Reward URL. This currently only contains a poolAddress. Let's only add the poolId for now (and not remove poolAddress), otherwise we will break the URLs that are out in the wild already.

Reply:
@GarfDev  I will make it part of this task in the next sprint: https://github.com/thxprotocol/api/issues/331 If you have done any work regarding this topic please commit it on a branch and let me know in the referenced issue.